### PR TITLE
145 fix bug proxy: stream terminated by RST_STREAM

### PR DESCRIPTION
**Pull request by MrWinstead**
* Apply patch suggested by @hpidcock at
https://github.com/improbable-eng/grpc-web/issues/145#issuecomment-371716140

### CODE REVIEW

**Comment by MrWinstead** [comment]
I'm currently using this as part of an internal project and the fix allows a javascript gRPC client to communicate from a ReactJS Redux action to a python 3 gRPC server.

File: go/grpcwebproxy/main.go [hunk 1]
```diff
@@ -104,7 +104,9 @@ func buildGrpcProxyServer(logger *logrus.Entry) *grpc.Server {
 	director := func(ctx context.Context, fullMethodName string) (context.Context, *grpc.ClientConn, error) {
 		md, _ := metadata.FromIncomingContext(ctx)
 		outCtx, _ := context.WithCancel(ctx)
-		outCtx = metadata.NewOutgoingContext(outCtx, md.Copy())
+    mdCopy := md.Copy()
```
Comment (jonahbron) [inline go/grpcwebproxy/main.go]: Some weird indentation going on here.

**Comment by jonahbron** [comment]
This is brilliant.  The error that CI is picking up is:

          ✖ should handle aborting a streaming response mid-stream with propagation of the disconnection to the server

But I seriously doubt this patch has caused that regression, I suspect that error was already there.  Can this be merged in soon so I can run my proxy on the latest code?

**Comment by MrWinstead** [comment]
Thank you! I've updated the indentation. At first I just made the quick edit in vi, but now I finally opened the poor thing in Goland to correct things

**Comment by jonahbron** [comment]
@MrWinstead Still looks off in the Github diff.  I think the surrounding code uses actual tab characters, while yours uses spaces.

**Pull request merged**